<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AreaSettingDialog</class>
 <widget class="QDialog" name="AreaSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>250</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Left</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Right</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Front</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="spinFront"/>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="spinLeft"/>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="spinRight"/>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="spinBack"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>旋转角</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="spinRotateAngle"/>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="spinVisionAngle"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>视角</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>180</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_3">
    <item row="1" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>TopLeftY</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>BottomRightX</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>BottomRightY</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>TopLeftX</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="spinRedTopLeftX">
      <property name="minimum">
       <double>-99.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="spinRedTopLeftY"/>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="spinRedBottomRightX">
      <property name="minimum">
       <double>-99.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="spinRedBottomRightY"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>176</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_4">
    <item row="1" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>TopLeftY</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>BottomRightX</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>BottomRightY</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>TopLeftX</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="spinBlueTopLeftX"/>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="spinBlueTopLeftY"/>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="spinBlueBottomRightX"/>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="spinBlueBottomRightY"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>FOV</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AreaSettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AreaSettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
